samples/client/petstore/c/unit-tests/manual-PetAPI.c

```diff
@@ -115,15 +112,26 @@ int main() {
 // Pet upload file Test
 	apiClient_t *apiClient3 = apiClient_create();
 	FILE *file = fopen("/opt/image.png", "r");
+	char *buff;
+	int read_size,len;
+	binary_t * data;
+	if (file) {
+	fseek(file,0,SEEK_END);
+	read_size = ftell(file);
+	rewind(file);
+	data->data = (char*) malloc(sizeof(char) * (read_size + 1));
+	data->len = fread(data->data,sizeof(char),read_size,file);
+	data->data[read_size] = '\0';
+	}
 	if(file != NULL) {
 		api_response_t *respo = PetAPI_uploadFile(apiClient3,
 		                                          EXAMPLE_PET_ID,
 		                                          "dec",
-		                                          file);
+		                                          data);
 
 		api_response_free(respo);
 		fclose(file);
 	} else {
-    apiClient_free(apiClient3);
+    //apiClient_free(apiClient3);
```

Comment:
I would suggest removing this line as it's commented out and people have no idea why it's left there.

Reply:
I updated it, as apiClient object has to be freed once the requirement is over. Modified all test samples accordingly.